cmd/cp-main.go

```diff
@@ -519,6 +539,11 @@ func mainCopy(ctx *cli.Context) error {
 	session.Header.CommandStringFlags["encrypt-key"] = sseKeys
 	session.Header.CommandStringFlags["encrypt"] = sse
 	session.Header.CommandBoolFlags["session"] = ctx.Bool("continue")
+
+	// Preserve functionality not supported for windows
+	if ctx.Bool("preserve") && runtime.GOOS != "windows" {
```

Comment:
We shouldn't add code like this, we should error out in checkSyntax() if this occurs and let the user know. Adding code like to avoid saving the flag while on windows platform is buggy coding